src/main/java/com/cloudbees/jenkins/plugins/amazonecs/pipeline/ECSTaskTemplateStepExecution.java

```diff
@@ -204,24 +204,16 @@ private ECSTaskTemplateCallback(ECSTaskTemplate taskTemplate) {
          */
         protected void finished(StepContext context) throws Exception {
             Cloud c = Jenkins.get().getCloud(cloudName);
-            String override = taskTemplate.getTaskDefinitionOverride();
             if (c == null) {
                 LOGGER.log(Level.WARNING, "Cloud {0} no longer exists, cannot delete task template {1}",
                         new Object[] { cloudName, taskTemplate.getTemplateName() });
                 return;
             }
             if (c instanceof ECSCloud) {
                 ECSCloud ecsCloud = (ECSCloud) c;
-                if (override != null){
```

Comment:
We not longer need to worry about override here as ECSService won't remove a task definition defined in a template as an override.